components/bounty-hero.tsx

```diff
@@ -26,90 +40,95 @@ export default function BountyHero() {
-                  {activeIssue?.mergeProposals?.length || 0}
-                </span>
-                <span className="caption-small text-white-40 text-uppercase">
-                  {activeIssue?.mergeProposals?.length === 1
-                    ? t("info.proposals_one")
-                    : t("info.proposals_other")}
-                </span>
-              </div>
-
-              {networkIssue?.creationDate && (
-                <DateLabel
-                  date={networkIssue.creationDate}
-                  className="text-white"
-                />
-              )}
-            </div>
-          </div>
-          <div className="col-2 d-flex align-items-center justify-content-center">
-            <PriceConversor
-              currentValue={activeIssue?.amount || 0}
-              currency={activeIssue?.token?.symbol || t("common:misc.token")}
-            />
+                  <div className="d-flex align-items-center">
+                    <Avatar
+                      className="me-2"
+                      userLogin={activeIssue?.creatorGithub}
+                    />{" "}
+                    <GithubInfo
+                      parent="hero"
+                      variant="user"
+                      label={["@", activeIssue?.creatorGithub].join("")}
+                    />
+                  </div>
+
+                  <span className="caption-small">
+                    {(activeIssue?.repository && (
+                      <GithubInfo
+                        parent="list"
+                        variant="repository"
+                        label={activeIssue?.repository?.githubPath}
+                      />
+                    )) ||
+                      ""}
+                  </span>
+
+                  <span className="caption-small text-ligth-gray text-uppercase">
+                    <Translation label={"branch"} />
+                    <span className="text-primary">:{activeIssue?.branch}</span>
+                  </span>
+                </div>
+
+                <div className="mt-3 pt-1 d-inline-flex align-items-center justify-content-md-start gap-20">
+                  <div>
+                    <span className="caption-small mr-1 text-white">
+                      {activeIssue?.working?.length || 0}
+                    </span>
+                    <span className="caption-small text-white-40 text-uppercase">
+                      {t("info.working")}
+                    </span>
+                  </div>
+
+                  <div>
+                    <span className="caption-small mr-1 text-white">
+                      {activeIssue?.pullRequests?.length || 0}
+                    </span>
+                    <span className="caption-small text-white-40 text-uppercase">
+                      {activeIssue?.pullRequests?.length === 1
+                        ? t("info.pull-requests_one")
+                        : t("info.pull-requests_other")}
```

Comment:
No need to do that, i18next can handle plurals.

https://www.i18next.com/translation-function/plurals